bg_mpl_stylesheet/bg_mpl_stylesheet

```diff
@@ -203,7 +203,7 @@ font.family          : arial
 ## small, medium, large, x-large, xx-large, larger, or smaller
 font.size            : 15
 #font.serif          : DejaVu Serif, Bitstream Vera Serif, Computer Modern Roman, New Century Schoolbook, Century Schoolbook L, Utopia, ITC Bookman, Bookman, Nimbus Roman No9 L, Times New Roman, Times, Palatino, Charter, serif
-#font.sans-serif     : DejaVu Sans, Bitstream Vera Sans, Computer Modern Sans Serif, Lucida Grande, Verdana, Geneva, Lucid, Arial, Helvetica, Avant Garde, sans-serif
+font.sans-serif      : cm
```

Comment:
I doubt this is the right way to do it.  We want to keep this somewhat generic font family in place, but select from it.  It looks as if below you select 'cm' from the font family sans-serif, so you just need to add cm to the family. Does that work?